hw/ip/aes/data/aes.hjson

```diff
@@ -30,6 +29,28 @@
       local:   "true"
     }
   ],
+  inter_signal_list: [
+    { name:    "keymgr_key",
+      type:    "uni",
+      act:     "rcv",
+      package: "keymgr_pkg",
+      struct:  "keymgr_key",
+      width:   "1"
+    }
+    { name:    "entropy",
+      type:    "req_rsp",
+      act:     "req",
+      package: "csrng_pkg",
+      struct:  "csrng_entropy",
+      width:   "1"
```

Comment:
Could you please take a look at the csrng interface on v1-bronze and revise the interface accordingly? I assume AES talks to CSRNG directly not to Entropy source. the new interface is `csrng_pkg::csrng_req/rsp_t`.

Reply:
Thanks @eunchan for pinging me. If I understand this correctly, the `csrng_req/rsp_t` structs in the new `csrng_pkg.sv` actually describe the more advanced application interface to be used by e.g. the key manager or the rng distribution network. Peripherals such as aes and otbn will not interface to the csrng directly but via the rng distribution network instead and thus use a simplified interface.

It would be great if @mwbranstad or @martin-lueker could clarify this please.